src/tools/imgtool/main.cpp

```diff
@@ -836,20 +844,27 @@ static const struct command cmds[] =
 	{ "listdriveroptions",  cmd_listdriveroptions, "<format>", 1, 1, 0 }
 };
 
-int CLIB_DECL main(int argc, char *argv[])
+
+// ----------------------------------------------------------------------
+
+int main(int argc, char *argv[])
 {
 	int i;
 	int result;
 	const struct command *c;
 	const char *sample_format = "coco_jvc_rsdos";
 	std::string cmdname = core_filename_extract_base(argv[0]);
 
+#ifdef _WIN32
+	_setmode(_fileno(stdout), _O_U16TEXT);
+#endif // _WIN32
+
```

Comment:
Does this do unexpected things if you redirect `stdout` to a file?  Like `mame -oslog -verbose | tee output.log` which I imagine we'd want to have in some ASCII superset encoding.